parsec/backend/asgi/__init__.py

```diff
@@ -20,7 +20,7 @@
 from parsec.backend.config import BackendConfig
 from parsec.backend.asgi.administration import administration_bp
 from parsec.backend.asgi.redirect import redirect_bp
-from parsec.backend.asgi.rpc import rpc_bp
+from parsec.backend.asgi.rpc import rpc_ano_bp, rpc_auth_bp
```

Comment:
Maybe we need better naming for those blueprints? I don't mind `rpc`, but `rpc_authenticated_blueprint` would look better IMO.

Reply:
done

Reply:
we don't need multiple blueprints here, `rpc_bp` is good enough

Blueprint is a tool to do deferred initialization on the app object, we don't use to do code separation (we split the code between files for that !)